pkg/operator/k8sutil/taints.go

```diff
@@ -18,7 +18,7 @@ package k8sutil
 
 import (
 	v1 "k8s.io/api/core/v1"
-	scheduler "k8s.io/kubernetes/pkg/scheduler/algorithm"
+	scheduler "k8s.io/kubernetes/pkg/scheduler/api"
```

Comment:
Would you also verify that the `WellKnownTaints` below capure all of the latest well-known taints? It's possible though unlikely that there were more added.

Reply:
Yes, verified that we have the same 9 taints.